transcom/mymove
# B 17850 error message

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/TeamRoom.mvc/Show/848240)

## Summary

All phone number error messages are updated to be descriptive and user-friendly.

When a user does not enter a phone number with valid formatting / characters:

"Please enter a valid phone number. Phone numbers must be entered as ###-###-####."

### How to test

1. Access the milmovelocal
2. Login as an exist user or create a new user
3. Enter an incorrect phone number so the error message appears
4. Verify error message is "Please enter a valid phone number. Phone numbers must be entered as ###-###-####."

### Frontend
